pg_failover_slots.c

```diff
@@ -455,7 +458,7 @@ make_sync_failover_slots_dsn(StringInfo connstr, char *db_name)
 	{
 		Assert(WalRcv);
 		appendStringInfo(connstr, "%s dbname=%s", WalRcv->conninfo,
-						 db_name ? db_name : "postgres");
+						 db_name ? db_name : pg_failover_maintenance_db);
```

Comment:
I would rather change this to use `pg_failover_maintenance_db` as parameter when calling `make_sync_failover_slots_dsn` instead of passing `NULL`

Reply:
Thanks, I changed it and added an Assert for good measure.